config/application.rb

```diff
@@ -144,6 +144,7 @@ def config.database_configuration
 
     # Our templates shouldn't start with 'discourse/templates'
     config.handlebars.templates_root = 'discourse/templates'
+    config.handlebars.raw_template_namespace = "Ember.TEMPLATES"
```

Comment:
In the latest version of `ember-handlebars-template`, the namespace was changed to default to `JST`. However, that is breaking our Ember resolver which uses `Ember.TEMPLATES`

Related commit in `ember-handlebars-template`: https://github.com/tricknotes/ember-handlebars-template/commit/b8cc2e75b9d5a45159d46540142ae50eceeec74d